tests/unit/core/test_evaluate_horner_polynomial.py

```diff
@@ -0,0 +1,15 @@
+"""Test of the evaluate_horner_polynomial function."""
+
+import numpy as np
+
+from pyrealm.core.utilities import evaluate_horner_polynomial
+
+
+def test_evaluate_horner_polynomial():
+    """Test the evaluate_horner_polynomial function."""
+    for deg in range(10):
```

Comment:
This is being super picky but it's good to have a bit more context in unit test.

```suggestion
    """Test the evaluate_horner_polynomial function.
    
    This checks that the normal form (a + bx + cx2 + dx3) and output from the
    horner form evaluation (a + x(b + x(c + dx))) are equivalent for powers up to 10.
    """
    for deg in range(10):
```